src/ext/java/org/opentripplanner/ext/transmodelapi/model/framework/BrandingType.java

```diff
@@ -3,44 +3,65 @@
-            .type(Scalars.GraphQLString)
-            .dataFetcher(env -> ((Branding) env.getSource()).getImage())
-            .build())
-        .build();
-  }
+        return GraphQLObjectType.newObject()
+                .name("Branding")
+                .field(GraphQLFieldDefinition.newFieldDefinition()
+                        .name("id")
+                        .type(Scalars.GraphQLID)
+                        .dataFetcher(env -> TransitIdMapper.mapEntityIDToApi(env.getSource()))
+                        .build())
+                .field(GraphQLFieldDefinition.newFieldDefinition()
+                        .name("name")
+                        .description("Full name to be used for branding.")
+                        .type(Scalars.GraphQLString)
+                        .dataFetcher(env -> ((Branding) env.getSource()).getName())
+                        .build())
+                .field(GraphQLFieldDefinition.newFieldDefinition()
+                        .name("shortName")
+                        .description("Short name to be used for branding.")
+                        .type(Scalars.GraphQLString)
+                        .dataFetcher(env -> ((Branding) env.getSource()).getShortName())
+                        .build())
+                .field(GraphQLFieldDefinition.newFieldDefinition()
+                        .name("description")
+                        .description("Description of branding.")
+                        .type(Scalars.GraphQLString)
+                        .dataFetcher(env -> ((Branding) env.getSource()).getDescription())
+                        .build())
+                .field(GraphQLFieldDefinition.newFieldDefinition()
+                        .name("url")
+                        .description("URL to be used for branding")
+                        .type(Scalars.GraphQLString)
+                        .dataFetcher(env -> ((Branding) env.getSource()).getUrl())
+                        .build())
+                .field(GraphQLFieldDefinition.newFieldDefinition()
+                        .name("image")
+                        .description("URL to an image be used for branding")
+                        .type(Scalars.GraphQLString)
+                        .dataFetcher(env -> ((Branding) env.getSource()).getImage())
+                        .build())
+                .field(GraphQLFieldDefinition.newFieldDefinition()
+                        .name("privateCode")
+                        .type(Scalars.GraphQLString)
+                        .description("Branding private code")
+                        .dataFetcher(env -> {
+                            PrivateCode privateCode = ((Branding) env.getSource()).getPrivateCode();
+                            if (privateCode != null) {
+                                return privateCode.getValue();
+                            } else {
+                              return null;
+                            }
```

Comment:
```suggestion
                            return privateCode == null ? null : privateCode.getValue();
```

Reply:
Currently we are not using presentation for anything at Skåntetrafiken. I have not included those fields in transmodel API,  just added them as OTP model. That way they can be easily exposed in the future. If you prefer to not have those fields in OTP model I can remove them.